spec/recurly/performance_obligation_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+
+describe PerformanceObligation do
+  let(:performance_obligation) {
+    stub_api_request(:get, "performance_obligations/6", "performance_obligations/show-200")
+    Recurly::PerformanceObligation.find '6'
+  }
+
+  let(:performance_obligations) {
+    stub_api_request(:get, "performance_obligations", "performance_obligations/index-200")
+    Recurly::PerformanceObligation.all
+  }
+
+  describe "#find" do
+    it "returns a PerformanceObligation" do
+      performance_obligation.must_be_instance_of(PerformanceObligation)
+    end
+
+    it "has correct attributes" do
+      expect(performance_obligation.id).must_equal('6')
+      expect(performance_obligation.name).must_equal('Over Time (Daily)')
+      expect(refute_nil(performance_obligation.created_at))
```

Comment:
I found this new matcher which is more intuitive: https://ruby-doc.org/stdlib-3.1.1/libdoc/minitest/rdoc/Minitest/Expectations.html#method-i-wont_be_nil

Reply:
Changed to `expect(performance_obligation.created_at).wont_be_nil`